editor/scene_tree_dock.h

```diff
@@ -91,7 +91,7 @@ class SceneTreeDock : public VBoxContainer {
 		TOOL_SCENE_CLEAR_INHERITANCE,
 		TOOL_SCENE_CLEAR_INHERITANCE_CONFIRM,
 		TOOL_SCENE_OPEN_INHERITED,
-
+		TOOL_TOGGLE_SCENE_UNIQUE,
```

Comment:
```suggestion
		TOOL_TOGGLE_SCENE_UNIQUE_NAME,
```
More clear about the purpose.